packages/flutter_tools/templates/app_shared/android-java.tmpl/build.gradle.kts.tmpl

```diff
@@ -5,14 +5,17 @@ allprojects {
     }
 }
 
-rootProject.buildDir = file("../build")
+val newBuildDir: Directory = rootProject.layout.buildDirectory.dir("../../build").get()
```

Comment:
I made the templates be slightly more readable than the version in tests/the lockfile generation script.

The latter version is sort of forced to be one line, so that it can conveniently be the same for Kotlin/Groovy (as `def`/`val` force to be Groovy/Kotlin specific), so that the lockfile generation script can be agnostic between the Groovy/Kotlin dsl while we still use both

Reply:
> I made the templates be slightly more readable than the version in tests/the lockfile generation script.
> 
> The latter version is sort of forced to be one line, so that it can conveniently be the same for Kotlin/Groovy (as `def`/`val` force to be Groovy/Kotlin specific), so that the lockfile generation script can be agnostic between the Groovy/Kotlin dsl while we still use both
One thing to consider: multiple files have the same content. Why not unify their usage?